sources/editor/Stride.GameStudio/Program.cs

```diff
@@ -49,362 +45,331 @@
 using MessageBoxButton = System.Windows.MessageBoxButton;
 using MessageBoxImage = System.Windows.MessageBoxImage;
 
-namespace Stride.GameStudio
+namespace Stride.GameStudio;
+
+public static class Program
 {
-    public static class Program
+    private static App app;
+    private static IntPtr windowHandle;
+    private static bool terminating;
+    private static Dispatcher mainDispatcher;
+    private static RenderDocManager renderDocManager;
+    private static readonly ConcurrentQueue<string> LogRingbuffer = new();
+
+    [STAThread]
+    public static void Main()
     {
-        private static App app;
-        private static IntPtr windowHandle;
-        private static bool terminating;
-        private static Dispatcher mainDispatcher;
-        private static RenderDocManager renderDocManager;
-        private static readonly ConcurrentQueue<string> LogRingbuffer = new ConcurrentQueue<string>();
-
-        [STAThread]
-        public static void Main()
-        {
-            AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
-            EditorPath.EditorTitle = StrideGameStudio.EditorName;
+        AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
+        EditorPath.EditorTitle = StrideGameStudio.EditorName;
```

Comment:
Now that `StrideGameStudio` is in `Stride.GameStudio.Helpers` you have to include that namespace, same goes for `StrideEditorPlugin` and co. Right now the gamestudio can't build.

Reply:
I guess should work now :v: